packages/utils/src/utils/types.ts

```diff
@@ -0,0 +1,188 @@
+interface Coin {
+  owner: string;
+  amount: string;
+  asset_id: string;
+}
+
+interface Message {
+  sender: string;
+  recipient: string;
+  nonce: string;
+  amount: string;
+  data: string;
+  da_height: string;
```

Comment:
Shouldn't we have types for these things stored somewhere already? 🤔

Reply:
We have something similar in [`resource.ts`](https://github.com/FuelLabs/fuels-ts/blob/master/packages/account/src/providers/resource.ts), but the types defined in this file are specifically related to the chain config and have different interfaces than `RawCoin`/`RawMessage` in `resource.ts`.